gprofiler/metadata/system_metadata.py

```diff
@@ -178,7 +181,7 @@ def get_static_system_info() -> SystemInfo:
         kernel_version=uname.version,
         system_name=uname.system,
         processors=cpu_count,
-        memory_capacity_mb=round(psutil.virtual_memory().total / 1024 / 1024),
+        memory_capacity_mb=round(psutil.virtual_memory().total / 1024 / 1024),  # type: ignore
```

Comment:
This line used to have `type ignore` and then we removed it. Now, why does it want that again?

Reply:
Yea, it's shit, `types-psutil` [doesn't specify a return type](https://github.com/python/typeshed/search?q=virtual_memory) for `virtual_memory()`, so now it warns:
```bash
gprofiler/metadata/system_metadata.py:184: error: Call to untyped function "virtual_memory" in typed context
```

Reply:
Blehh